templates/zerver/integrations.html

```diff
@@ -1931,19 +1931,79 @@
 
     <div id="trello" class="integration-instructions">
 
+      <p>
+        First, create the stream you'd like to use for Trello notifications,
+        and subscribe all interested parties to this stream. We recommend the
+        name <code>trello</code>.
+      </p>
 
-      <p>First, create the stream you'd like to use for Trello notifications,
-      and subscribe all interested parties to this stream. We recommend the
-      name <code>trello</code>.</p>
+      <p>
+        Your webhook URL is:<br/>
+        <code>https://api.zulip.com/v1/external/trello?api_key=abcdefgh&stream=trello</code><br/>
+        where <code>api_key</code> is the API key of your Zulip bot,
+          and <code>stream</code> is the stream name you want the notifications sent to.
+      </p>
+
+      <p>
+        Before you'll be able to create a Webhook,
+        you need to follow steps below to get an <b>APPLICATION_KEY</b> and a <b>UserToken</b>.
+        Also we need to fetch the <b>idModel</b> of the board.
+      </p>
+
+      <p>
+        To generate the <b>APPLICATION_KEY</b> open this URL in your web browser:<br/>
+        <code>https://trello.com/1/appkey/generate</code>.
+      </p>
 
-      <p>Next, download a copy
-      of <a href="https://github.com/zulip/trello-to-zulip">trello-to-zulip</a>
-      and follow the instructions in <code>README.md</code>. When you
-      make changes in Trello, they will be reflected in Zulip:
+      <p>
+        To generate read access token, fill in and open this URL in the browser while being logged into your Trello account:
+        <code>https://trello.com/1/authorize?key=&ltAPPLICATION_KEY&gt&name=Issue+Manager&expiration=never&response_type=token&scope=read</code><br/>
+        You will receive the <b>UserToken</b>, save it.
       </p>
 
-      <img class="screenshot" src="/static/images/integrations/trello/001.png" />
-      <p><small>Thanks to Nathan from FoundationDB for <a href="https://github.com/nathanlws/trello-to-zulip">submitting this integration</a>!</small></p>
+      <p>
+        From board URL get <b>TRELLO_BOARD_SHORT_ID</b>.
+        URL should look like:<br/>
+        <code>https://trello.com/b/TRELLO_BOARD_SHORT_ID/boardName</code>.
+      </p>
+
+      <p>
+        Now you have: <b>APPLICATION_KEY</b>, <b>UserToken</b>, <b>TRELLO_BOARD_SHORT_ID</b>.<br/>
+        Go to your web browser and open filled URL:<br/>
+        <code>https://api.trello.com/1/board/&ltTRELLO_BOARD_SHORT_ID&gt?key=&ltAPPLICATION_KEY&gt&token=&ltUserToken&gt</code><br/>
+        From returned JSON get <b>id</b> value. It's your <b>idModel</b>, save it.
+      </p>
+
+      <p>
+        In a last step you need to create your webhook.
+        You need to send POST request to following URL with data:
+        <code>https://api.trello.com/1/tokens/&ltUserToken&gt/webhooks/?key=&ltAPPLICATION_KEY&gt</code><br/><br/>
+        <code>
```

Comment:
Yeah, it's definitely no fun to be hand-coding that sort of HTML encoding like this.  I think probably as we work on making webhooks have a smoother installation process, a lot of these docs can be standard (shared between all the integrations rather than written by hand for each one) and we should be able to fix this nonsense too.